src/utils/checkRight.ts

```
@@ -0,0 +1,32 @@
+import { Response } from 'express';
+import { AuthenticatedRequest } from '../types/auth';
+import { prisma, Role } from '../config/prisma';
+
+export const checkRight = async (req: AuthenticatedRequest<{}, {}, {}>, res: Response) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)